Lib/test/test_pathlib.py

```diff
@@ -2298,5 +2298,44 @@ def check():
             check()
 
 
+class CompatiblePathTest(unittest.TestCase):
+    """
+    Test that a type can be made compatible with PurePath
+    derivatives by implementing division operator overloads.
+    """
+
+    class CompatPath:
+        """
+        Simple class to store a string and uses combine that
```

Comment:
"uses combine"? This sentence doesn't read right to me.

Reply:
Agreed, I rewrote it to make sense. Thanks for pointing it out.